openid_connect_configuration.conf

```diff
@@ -43,6 +43,16 @@ map $host $oidc_hmac_key {
     default "ChangeMe";
 }
 
+map $host $zone_sync_timeout {
```

Comment:
May be 'skew' or 'leeway'?

Reply:
I agree, "zone_sync_leeway" looks clearer.